src/Analyzer.Utilities.UnitTests/JsonLineNumberResolverTests.cs

```diff
@@ -265,6 +266,233 @@ public void ResolveLineNumber_ReturnsCorrectLineNumber(string path, object[] pat
             Assert.AreEqual(expectedLineNumber, resolvedLineNumber);
         }
 
+        [DataTestMethod]
+        [DataRow("resources[0].type", 11, DisplayName = "")]
+        [DataRow("resources[0].name", 12, DisplayName = "")]
+        [DataRow("resources[0].properties.somePath", 14, DisplayName = "")]
+        public void ResolveLineNumber_ContextWithOffset_ReturnsCorrectLineNumber(string path, int lineNumber)
+        {
+            TemplateContext templateContextWithOffset = new()
```

Comment:
Most of the information here is irrelevant to the test.  Please trim it down to only what's necessary.

Reply:
Instead of using two identical contexts, differing only by the offset, you can create a context with 0 offset, get the line number, then update the offset and get the line number again.

Reply:
No longer using line numbers